contrib/babelfishpg_common/sql/upgrades/babelfish_common_helper--5.1.0--5.2.0.sql

```diff
@@ -44,5 +44,37 @@ BEGIN
 END
 $body$;
 
+CREATE OR REPLACE FUNCTION sys.int4varbinary(INT4, integer, boolean)
+RETURNS sys.BBF_VARBINARY
+AS 'babelfishpg_common', 'int4varbinary'
+LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION sys.float4varbinary(REAL, integer, boolean)
+RETURNS sys.BBF_VARBINARY
+AS 'babelfishpg_common', 'float4varbinary'
+LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
```

Comment:
we are not making any changes to these functions, why are we creating or replacing it here ?